AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -277,9 +277,15 @@ - (void)deleteLogsWithBatchId:(NSString *)batchId groupId:(NSString *)groupId {
 
     // Deserialize target token.
     NSString *encryptedToken = row[self.targetTokenColumnIndex];
-    if (![encryptedToken isKindOfClass:[NSNull class]]) {
+    if (![encryptedToken isKindOfClass:[NSNull class]] && (encryptedToken.length > 0)) {
       NSString *targetToken = [self.targetTokenEncrypter decryptString:encryptedToken];
-      [log addTransmissionTargetToken:targetToken];
+      if (targetToken) {
+        [log addTransmissionTargetToken:targetToken];
+      } else {
+        MSLogWarning([MSAppCenter logTag], @"Failed to decrypt the target token.");
```

Comment:
Just warning message? I think we should drop the log with `error` message.

Reply:
I agree.